### PR TITLE
Feature/137005175/notification

### PR DESCRIPTION
**Pull request by olawalejarvis**
### What does this PR do?

Send SMS alert to buyers  for products ordered, and wallet activities.
Display notifications on the user dashboard.

### How should this be manually tested?
Register a user and place an order
Deposit money into wallet account
Pay for order from wallet account

What are the relevant pivotal tracker stories?

#137005175

### Screenshot(s)
oie_sg7xgzfavbrl

### CODE REVIEW

File: server/methods/core/wallet.js [hunk 1]
```diff
@@ -28,18 +38,41 @@ Meteor.methods({
...
           from: sender.emails[0].address,
           date: new Date(),
           transactionType: "Credit"
         });
+      } else {
+        notification = {
+          userId: userId,
+          name: "Debit Transaction",
+          type: transactionType,
+          message: `Debit Alert! ${amount} naira was deducted from your account for the payment of the order you made; '\n' Order Id: ${transactions.orderId} '\n' Date: ${transactions.date}`,
```
[the remark this answers is not in the document]
Comment (olawalejarvis) [reply server/methods/core/wallet.js]: Fixed

File: server/methods/core/orders.js [hunk 2]
```diff
@@ -893,5 +1015,27 @@ Meteor.methods({
...
+          cmd: "sendquickmsg",
+          owneremail: Meteor.settings.SMS.OWNEREMAIL,
+          subacct: Meteor.settings.SMS.SUBACCT,
+          subacctpwd: Meteor.settings.SMS.SUBACCTPWD,
+          message: smsContent.message,
+          sender: "ANBU-SQUAD",
+          sendto: smsContent.to,
+          msgtype: 0
+        }
+      }, (error, result) => {
```
[the remark this answers is not in the document]
Comment (olawalejarvis) [reply server/methods/core/orders.js:1036]: Fixed

**Review by andela-aolaniran — COMMENTED** [review]
Looks good. But the review app isn't working on heroku ?

File: imports/plugins/core/checkout/client/templates/checkout/payment/methods/cards.js [hunk 3]
```diff
@@ -100,7 +100,7 @@ Template.walletPayment.events({
         Meteor.call("cart/submitPayment", paymentMethod);
         Alerts.toast("Payment Successful", "success");
       } else {
-        Alerts.toast("An error occured, please try again", "error");
+        Alerts.toast("An error occured, please try again, I am the wallet", "error");
```
Comment (vonvick) [inline imports/plugins/core/checkout/client/templates/checkout/payment/methods/cards.js:103]: Any reason as to why you have " I am the wallet" in the toast?

**Review by vonvick — COMMENTED** [review]
LGTM

File: imports/plugins/core/checkout/client/templates/checkout/payment/methods/cards.js (hunk 3 above)
Comment (olawalejarvis) [reply imports/plugins/core/checkout/client/templates/checkout/payment/methods/cards.js:103]: Fixed

**Pull request merged**